docs/release_notes.adoc

```diff
@@ -6,6 +6,7 @@ include::include.adoc[]
 === What's New In This release
 
 * The `@Retry` extension can now be configured to evaluate a `condition` (<<extensions.adoc#_retry,Docs>>)
+* Successful retries now also make a test pass when using `Retry.Mode.FEATURE` or `Retry.Mode.SETUP_FEATURE_CLEANUP`.
```

Comment:
Prefix with `Fix`, the previous line regarding condition should start with `Add`

Reply:
Fixed.